package.json

```diff
@@ -12,6 +12,7 @@
   },
   "type": "module",
   "workspaces": [
+    "examples/*",
```

Comment:
Might be good to do this and the bing maps fix in separate PRs. This one in particular has caused some issues in the past. Make sure you can build the website. May have to revert if it messes up publishing.